### ISSUE TITLE
Ability to Commit Back to Git from Bolt

### ISSUE THREAD

**Issue by kevinnkansah** [stackblitz/bolt.new#327]
When importing a git repo into bolt then making changes in bolt and opening in Stackblitz, Stackblitz should be able to recognise that this is an imported git repo and allow the user to commit and not show "Create Git Repo" This is bad because the only way to save my changes is to create another repo suffixed by "v2" etc.

**Comment by plugmakr** [comment]
This would be fantastic. Pushing directly to Git from Bolt is ideal.

**Comment by Faaiqster** [comment]
Would really be a huge update if this feature was implemented. I'm finding it can only create new repos each time and won't update the existing repo or connected to which really defeats the purpose of the repo

**Comment by vijaytilak** [comment]
Yeah this is a much needed feature for developing with Bolt

**Comment by tsimms** [comment]
+1 here. in bolt, it would be nice if there were a [Commit] button you could push, when things look right -- bolt could fill in a nice commit msg too -- for committing changes to local repo with 1-click.

**Comment by rsweetland** [comment]
Agreed. I think Glitch implemented this well. As I recall, they exposed the git binary in their terminal. A Glitch instances could be a remote. it could also push to other remotes. It played well various local / remote dev workflows.
